### ISSUE TITLE
nmpbox fails with namespaced package

### ISSUE THREAD

**Issue by GROpenSourceDev** [arei/npmbox#67]
When I run npmbox @angular/benchpress I get the following error. Looks like the '/' in the package name is causing problems for the rack function. 

```
C:\Users\homeworking\AppData\Roaming\npm>npmbox @angular/benchpress 

Boxing @angular/benchpress ...
fs.js:640
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open 'C:\Users\homeworking\AppData\Roaming\npm\.npmbox.cache\@angular\benchpress.npmbox'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.writeFileSync (fs.js:1333:33)
    at rack (C:\Users\homeworking\AppData\Roaming\npm\node_modules\npmbox\npmboxxer.js:224:7)
    at extractPackageName (C:\Users\homeworking\AppData\Roaming\npm\node_modules\npmbox\npmboxxer.js:261:5)
    at next (C:\Users\homeworking\AppData\Roaming\npm\node_modules\npmbox\npmboxxer.js:203:9)
```

This looks to apply to all @namespace/packagename npm packages. Perhaps npmbox needs to rack under a different name, e.g @angular#benchpress that does not get resolved into an additional folder. 

Inside the cache though we do want to maintain the @angular/benchbress folder structure as this is what you get when just doing a vanilla npm install.

Run with npmbox v4.0.1

**Comment by danfuzz** [comment]
I believe this is fixed by PR #68 (currently awaiting comments / approval / merge).

**Comment by danfuzz** [comment]
BTW, I merged PR #68 into PR #71 and closed the former.

**Comment by danfuzz** [comment]
Closing this as presumed-fixed by v4.0.2 (just published). During ad-hoc testing of the version I in fact succeeded at boxing `@angular/benchpress`, but YMMV of course!

Please comment / reopen if you find that the new version is still causing you trouble.
